event_listener/dfusion_db/snapp_event_receiver.py

```diff
@@ -25,9 +25,8 @@ def save(self, event: Dict[str, Any], block_info: Dict[str, Any]) -> None:
     def save_parsed(self, deposit: Deposit) -> None:
         try:
             self.database.write_deposit(deposit)
-        except AssertionError as exc:
-            logging.critical(
-                "Failed to record Deposit [{}] - {}".format(exc, deposit))
+        except Exception as exc:  # pragma: no cover
```

Comment:
I feel like all these `no cover` are a bit like cheating. Why should we not test that we catch all types of execptions when processing a deposit?